RecDP/pyrecdp/primitives/operations/text_reader.py

```diff
@@ -23,28 +23,29 @@ def __init__(self, input_dir = ""):
         
     def process_rayds(self, ds=None) -> Dataset:
         import ray.data as rd
-        def convert_json(s):
-            if isinstance(s, str):
-                content = json.loads(s)
-            elif isinstance(s, dict):
-                content = json.loads(s['text'])
-            if 'meta' in content:
-                content['meta'] = str(content['meta'])
-            return content
-        self.cache = rd.read_text(self.input_dir).map(convert_json)
+        self.cache = rd.read_json(self.input_dir)
```

Comment:
@haojinIntel , maybe we should also use some mixed schema jsonl to test? I had encountered with ray read error when jsonl contains nested scheme, that is why I did some manual fix in above